packages/app/src/App.tsx

```diff
@@ -69,16 +57,14 @@ const AppRouter = app.getRouter();
 const routes = (
   <FlatRoutes>
     <Navigate key="/" to="catalog" />
-    <Route path="/catalog" element={<CatalogIndexPage initiallySelectedFilter="all" />} />
+    <Route path="/catalog" element={<CatalogIndexPage />} />
```

Comment:
can you re-add the filter it was there intentionally

```suggestion
    <Route path="/catalog" element={<CatalogIndexPage initiallySelectedFilter="all" />} />
```